perf/run-benchmarks

```diff
@@ -379,9 +390,18 @@ def run_benchmarks(
                     results_msgs[-1] += " WARNING: more errors than std"
 
     # Show summary data
-    for msg in results_msgs:
-        print(msg)
+    print("\n".join(results_msgs))
+
+    if summary_file_path:
+        with chdir(called_dir):
+            summary_file = open(summary_file_path, "w+")
+            summary_file.write(durations)
+            summary_file.close()
+
     if plot_benchmarks:
+        import matplotlib.pyplot as plt
```

Comment:
sometimes Python is convenient :)

Reply:
btw this package needs to be installed to unbreak the daily benchmarks: https://app.circleci.com/pipelines/github/returntocorp/semgrep/3265/workflows/b09dbf63-092e-4626-aaec-a9af8eec4e4d/jobs/3878

Reply:
hm weird on github actions hiding it behind the if was fine

Reply:
Also @mjambon are we already running the benchmarks nightly? I'm not seeing on the dashboard runs after 4/4 (other than the ones I did yesterday and today), but that might be when I broke it with matplotlib

Reply:
I've been working on running the benchmarks nightly using github actions, so I might have duplicated work

Reply:
Looking at the CircleCI benchmarks, are they running the latest run-benchmarks? The line number highlighted indicates that the import is still at the top of the file

Reply:
If you navigate to the [list of jobs in CircleCI](https://app.circleci.com/pipelines/github/returntocorp/semgrep?branch=develop), you can see the matplotlib error is gone, but there's [another problem](https://app.circleci.com/pipelines/github/returntocorp/semgrep/3352/workflows/6941e8d2-5516-478d-b9ea-3a3952df1aed/jobs/3964).

Reply:
hm, not sure why this one is happening---is there an easy command to reproduce this locally?